src/plugins/Makefile.am

```diff
@@ -276,8 +276,15 @@ sources += tlsmon/tlsmon.h
 sources += tlsmon/private.h
 endif
 
+if PLUGIN_HYPERBEE
+sources += hyperbee/hyperbee.cpp
+sources += hyperbee/hyperbee.h
+sources += hyperbee/private.h
+sources += filesystem.hpp
```

Comment:
Adding `filesystem.hpp` here is not necessary

Reply:
Removed